discord/asset.py

```diff
@@ -120,6 +122,15 @@ def __repr__(self):
 
         Raises
         --------
+        DiscordException
+            The URL or :class:`ConnectionState` was not provided.
+
+            .. note::
+
+                :class:`PartialEmoji` will not have a state if you make
+                your own instance via ``PartialEmoji(animated=False, name='x', id=2345678)``.
```

Comment:
This note seems misplaced. The example is also superfluous since it's implied by stating "create your own instance", everyone should know how a class is created.

I'm not sure if this note is a good idea in general really.